django/contrib/gis/db/backends/oracle/schema.py

```diff
@@ -31,6 +31,11 @@ def __init__(self, *args, **kwargs):
     def geo_quote_name(self, name):
         return self.connection.ops.geo_quote_name(name)
 
+    def quote_value(self, value):
+        if isinstance(value, self.connection.ops.Adapter):
+            return super().quote_value(str(value))
+        return super().quote_value(value)
```

Comment:
I decided to fix `quote_value()` anyway, in order not to mask the original exception:
```
ORA-00904: "SDO_EQUAL": invalid identifier
```
with
```
ORA-00907: missing right parenthesis
```